ndt7-client-cc.cpp

```diff
@@ -85,6 +85,24 @@ void BatchClient::summary() noexcept {
   std::cout << summary.dump() << std::endl;
 }
 
+struct KV {
+  std::string first;
+  std::string last;
+};
+
+static bool token(std::string s, std::string sep, KV& kv) {
```

Comment:
```suggestion
// getNextToken returns true if the string @p s contains @p sep and false otherwise. On successful
// return, the @p kv parser state is mutated and will contain the first token in its .first entry and the rest
// of the string yet to parse into its .rest entry.
static bool getNextToken(std::string s, std::string sep, KV &kv) {
```